setup.py

```diff
@@ -14,9 +14,9 @@
 
 VERSION='0.2.5'
 
-RE_BLANCK=re.compile(r"^\s*#?")
+RE_COMMENT = re.compile(r"^\s*#")
 with (Path(__file__).parent / 'requirements.txt').open() as f:
-    requirements = [x for x in f.read().splitlines() if not RE_BLANCK.match(x)]
+    requirements = [x for x in f.read().splitlines() if not RE_COMMENT.match(x) and x]
```

Comment:
du coup, le `and x` n'est plus nécessaire

Reply:
Oui, ça fonctionne aussi.